### DISCUSSION TITLE
The demo of Unitree humanoid robot

### DISCUSSION THREAD

**Discussion by Guanbin-Huang** [isaac-sim/IsaacLab#160] [Q&A] [answered]
I recently came across a demo related to Isaac Sim with a humanoid robot that you showcased yesterday. I'm quite interested in studying it further. Could you kindly provide me with information regarding the whereabouts of this demo? I would greatly appreciate it.

I found a YouTube video (https://www.youtube.com/watch?v=bx4l3daS-Q8&t=199s) where the timestamp 49:02 suggests the presence of the demo. Is this the correct video? If not, could you please point me in the right direction?
<img width="798" alt="image" src="https://github.com/NVIDIA-Omniverse/Orbit/assets/30463291/40eafd3f-735d-42c5-a49c-06a394c185f5">

**Comment by Mayankm96** [comment, marked as answer]
Hi, yes, we plan to put this as a part of our legged robots suite in Orbit. Currently, we are waiting for a confirmation from Unitree to share the robot asset publicly :)

**Reply by Guanbin-Huang** [reply]
Great, really wanna use your stunning work as my introductory tool to learn reinforcement learning.

**Reply by Guanbin-Huang** [reply]
By the way, would and coudld you release the code for cassie or other open source human robot or biped robot in advance?  So that even without the confirmation, we still have the opportuniies to have a try on the "Tesla Optimus" like robot.

**Reply by Mayankm96** [reply]
The environments for various legged robots have been included in devel: https://github.com/NVIDIA-Omniverse/Orbit/tree/main/source/extensions/omni.isaac.orbit_tasks/omni/isaac/orbit_tasks/locomotion/velocity/config
